src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -43,17 +45,22 @@ type Props = {
 };
 
 const headerStyles = css`
-	${until.mobileLandscape} {
-		padding-left: 10px;
-	}
-	${until.tablet} {
-		padding-top: 33px;
-	}
```

Comment:
I was finding it a little tricky to make tweaks with the mix of `from` and `until` so did a little refactor here to just use `from`